app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -266,8 +267,11 @@ class NavigationOverlayFragment : Fragment() {
             .subscribe { accountState ->
                 when (accountState) {
                     is AccountState.AuthenticatedWithProfile -> {
-                        accountState.profile.avatarSetStrategy.invoke(fxaButton)
+                        accountState.profile.avatarSetStrategy
+                            .setTransformation(RoundCornerTransformation(fxaButton.width.toFloat()))
+                            .invoke(fxaButton)
                         fxaButton.contentDescription = resources.getString(R.string.fxa_navigation_item_signed_in2)
+                        fxaButton.imageTintList = null
```

Comment:
Do we want the image tint list in other blocks? If not, can we remove this in the XML instead of here?

Reply:
Yes, we need it on the other buttons.

Reply:
I was unclear, by other blocks I meant the other conditions of the when expression.  Right now it's going to keep its tint list until eventually going through `AccountState.AuthenticatedWithProfile`, but then lose it and never regain it.  So any of the other states could look two different ways depending on the flow that the user goes through.

Instead of nulling this out here, can we null it in the XML so that it's always the same for this button?

Reply:
I think that we have to do it in code, because the style on the button applies its own tint. I tested signing in and signing out and the icon colour doesn't change so I think this line is okay? Adding back any tints can be taken care of in the follow-up since it's more related to the icons rather than the profile image.

Reply:
I believe all we need to do is add `android:tint="@null"` to the button.  iirc, the more specific rule (on the view) will override the more general one (the style).

More generally, this is safe _right now_, since the style isn't changing anything we particularly care about, but that could change if the style were ever updated.  We've taken great pains to make sure that the data and the UI map 1:1, so it's worth putting in extra effort to avoid creating a situation where stale view state winds up changing our UI.